tests/test_compiled_messages.py

```diff
@@ -1,59 +0,0 @@
-"""
```

Comment:
Great question. The tests never ran when I tried on master, because ``CONFIGURATION.translated_locales`` returns ``[]``. From reading the tea leaves, I think this test made sense when run in edx-platform, but stopped doing anything when moved into its own repo.